lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - any argument
+// Result: argument x
+const id = x => x;
+
+// Async identity function
+//   x - any argument
+//   callback - function, (err, data)
+const asyncId = (x, callback) => {
+  callback(null, x);
+};
+
+// Mapable object iteration identity function
+//   items - array
+//   fn - function
+//   callback - function, (err, data)
+const mapIterationId = (items, fn, callback) => {
```

Comment:
You don't seem to be using the second `fn` argument, why have you added it to the function's signature?

Reply:
I thought there should be the same signature as in other functions, iterating over arrays